docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -48,7 +48,7 @@ k8s-cluster-101-md-0-577b7dc766-p2gkz   Ready      <none>   2d20h   v1.18.6+vmwa
 
 ##### Oracle Container Registry
 
-You will need an Oracle account. The following steps will direct you to accept the license agreement for WebLogic Server.  Make note of your Oracle Account password and email.  This sample pertains to 12.2.1.4, but other versions may work as well.
+You will need an Oracle Container Registry account. The following steps will direct you to accept Oracle Standard Terms and Restrictions to pull the WebLogic Server images.  Make note of your Oracle Account password and email.  This sample pertains to 12.2.1.4, but other versions may work as well.
```

Comment:
will direct you to accept Oracle Standard Terms -> will direct you to accept the Oracle Standard Terms

Reply:
fixed